lib/backend.js

```diff
@@ -12,18 +12,18 @@ const defaults = {
 	prefix : ''
 };
 
-function SequelizeBackend (db, options){
+function SequelizeBackend (db, options, Sequelize){
```

Comment:
Previously Sequelize class is available on the object prototype (db.Sequelize)
so we don't need to change the API of this method